sklearn/preprocessing/data.py

```diff
@@ -250,9 +260,22 @@ def fit(self, X, y=None):
         if feature_range[0] >= feature_range[1]:
             raise ValueError("Minimum of desired feature range must be smaller"
                              " than maximum. Got %s." % str(feature_range))
-        data_min = np.min(X, axis=0)
-        data_range = np.max(X, axis=0) - data_min
+
+        if len(X.shape) == 2:
```

Comment:
The input array is allowed to be 2-d. The call to `check_arrays` on line 268 of the most recent commit has `ensure_2d=False`.

https://github.com/scikit-learn/scikit-learn/pull/5002/files#diff-5ebddebc20987b6125fffc893f5abc4cR268